src/blog/2024/01/soc2.md

```diff
@@ -0,0 +1,57 @@
+---
+title: FlowFuse is now SOC 2 Type 1 Compliant
+subtitle: FlowFuse's Path to SOC 2 Type 1 Compliance - A Testament to Our Commitment to Securing Customer and User Data.
+date: 2024-01-15
+authors: ["grey-dziuba"]
+image: /blog/2024/01/images/soc-2-compliant.png
+tags:
+    - posts
+    - flowfuse
+    - news
+---
+
+
+SOC 2, governed by the American Institute of Certified Public Accountants (AICPA), is a crucial framework for organizations handling customer data. It involves an independent audit assessing whether a company's controls are effectively designed and operational. Achieving SOC 2 compliance is not just about meeting standards; it's a declaration of our unwavering commitment to data security and customer trust.
+
+<!--more-->
+
+# Improving Your Security Posture
+For FlowFuse, pursuing SOC 2 compliance was a natural step in enhancing our security posture. In today's landscape, where cybersecurity threats and data breaches are rampant, prioritizing information security is not optional but a necessity. Our SOC 2 audit was more than a procedural hurdle; it was a comprehensive validation of our robust controls and processes by an independent third party.
+
+# Why We Pursued SOC 2 Now
+As we continue to grow and evolve, ensuring the security of our systems and data becomes ever more critical. Our decision to pursue SOC 2 compliance now reflects our proactive stance on security and our dedication to data protection, aligning perfectly with our goal to "Build bespoke, flexible, and resilient manufacturing applications."
+
+# FlowFuse's Journey to SOC 2 Compliance
+## Compliance Partners
+### Vanta:
+Our partnership with Vanta, a leader in Trust Management, was instrumental. Vanta's tools automated the collection of our audit evidence, laying a strong foundation to protect customer data.
+
+### Advantage Partners:
+Working with Advantage Partners, our audit firm, was a seamless experience. Their expertise and support were vital in our efficient, successful attainment of SOC 2 compliance.
+
+## Process
+The journey to SOC 2 compliance, while extensive, was streamlined thanks to our partners. Vanta's integration with our systems and guidance on policies and procedures expedited our readiness. Following this, Advantage Partners evaluated our controls during our Type I audit and subsequently issued our report.
+
+## Timeline
+Achieving compliance is a significant undertaking. However, with dedicated focus and the right partners, we were able to fast-track our readiness from months to weeks. This preparation not only facilitated our current audit but also laid a foundation for more streamlined future audits.
```

Comment:
```suggestion
The independent audit was performed by Advantage Partners. Their expertise played a large role in our successful attainment of this certification. Before the audit was performed the company went through an extensive process to uncover what policies were missing, required updating, or were already in place. Further, lots of tribal knowledge has been written down and is now enforced by internal policies. For example

1. Handbook link 1
1. Handbook link 2
1. Handbook link 3

It's been a team effort from engineering to updated HR polices!
```

Reply:
@gdziuba As the diff says; it's all in the handbook. https://flowfuse.com/handbook/company/security/

Pick 3 of the most interesting ones I'd say!

Reply:
Lol.  I guess I interpreted the comment as specific SOC 2 changes.  Got it.  Thanks for the link and direction.

Reply:
```suggestion
The independent audit was performed by Advantage Partners. Their expertise played a large role in our successful attainment of this certification. Before the audit was performed the company went through an extensive process to uncover what policies were missing, required updating, or were already in place. Further, lots of tribal knowledge has been written down and is now enforced by internal policies. For example

1. [Data Management Policy](https://flowfuse.com/handbook/company/security/data-management/#data-management-policy)
2. [Access Control Policy](https://flowfuse.com/handbook/company/security/access-control/#access-control-policy)
3. [Incident Response Policy](https://flowfuse.com/handbook/company/security/incident-response/#incident-response-plan)
4. [Human Resources Security Policy](https://flowfuse.com/handbook/company/security/human-resources/#human-resources-security-policy)

It's been a team effort from engineering to updated HR polices!
```